src/main/java/com/google/sps/servlets/AccountData.java

```diff
@@ -74,7 +97,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     Entity userEntity = new Entity("User");
 
     // adds data to userentity
-    userEntity.setProperty("id", userService.getCurrentUser().getUserId());
+    userEntity.setProperty("ID", userService.getCurrentUser().getUserId());
```

Comment:
Use a constant for "ID".

Reply:
What is this constant for?

Reply:
Literal strings are like magic numbers, it's usually good to define them as constants instead of littering them in code, which is hard to track down and update.